### DISCUSSION TITLE
请教一下，DataType 上没有这些属性是如何实现绑定的

### DISCUSSION THREAD

**Discussion by ErickWei** [irihitech/Ursa.Avalonia#244] [Q&A]
如图所示，在导航菜单样例中，DateType 上没有这些属性，又是如何实现绑定的？

<img width="1165" alt="image" src="https://github.com/irihitech/Ursa.Avalonia/assets/62826892/efbbc0af-9fcb-4dcf-993f-3b4832d968d4">

**Comment by rabbitism** [comment]
在属性上标注[InheritDataTypeFromItems]属性可以将他的数据类型上下文变成ItemsSource中的元素类型
